sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/SchemaIOTableProviderWrapper.java

```diff
@@ -80,7 +80,7 @@ public BeamSqlTable buildBeamSqlTable(Table tableDefinition) {
     }
   }
 
-  private BeamTableStatistics getTableStatistics(PipelineOptions options) {
+  public BeamTableStatistics getTableStatistics(PipelineOptions options, SchemaIO schemaIO) {
```

Comment:
nit: I think this just needs to be protected so that implementors can override
```suggestion
  protected BeamTableStatistics getTableStatistics(PipelineOptions options, SchemaIO schemaIO) {
```